lib/util/git/index.ts

```diff
@@ -143,6 +144,41 @@ let gitInitialized: boolean;
 
 let privateKeySet = false;
 
+export const GIT_MINIMUM_VERSION = '2.33.0'; // git show-current
+
+export async function validateGitVersion(): Promise<boolean> {
+  let version: string;
+  const globalGit = Git();
+  try {
+    const raw = await globalGit.raw(['--version']);
+    for (const section of raw.split(/\s+/)) {
+      if (semver.isVersion(section)) {
```

Comment:
You need to coerce before: `git version 2.34.1.windows.1`

Check the ci init, there we log the git version.

https://github.com/renovatebot/renovate/blob/main/tools/check-git-version.mjs#L10-L14

Reply:
Hi @viceice is it fine if I use `semver-coerced` instead of `semver`? It does coercion by default.

Reply:
Sure, you can try it.